### DISCUSSION TITLE
Status of Glut dependency

### DISCUSSION THREAD

**Discussion by HerrNamenlos123** [openframeworks/openFrameworks#7672] [General]
This is related to #7524. I am asking what the current status of Glut is. It caused some trouble in CMake as it is somehow not needed and not available in the apothecary, on the other hand it seems to be needed for SOME parts, and it really is not clear. Someone else said it is more or less deprecated and subject to be removed, someone else reported their build failed due to glut errors.

What is the status? Can it be removed, and if it is needed, what for and under which circumstances?

**Comment by NickHardeman** [comment]
It doesn't seem like it is being used for the default windows for any platform as far as I can tell. Outside of ofAppGlutWindow I can't seem to find any references. 

https://github.com/openframeworks/openFrameworks/blob/1b90592b26d379d5bee162f9c93665cf85d3ba9f/libs/openFrameworks/app/ofMainLoop.cpp#L14-L30

**Reply by HerrNamenlos123** [reply]
Thanks for the reply, from my perspective that means i can remove it from the packages

**Comment by NickHardeman** [comment]
pinging @ofTheo for some more insight

**Comment by ofTheo** [comment]
I think we can definitely pull it at this point. 
It hasn't been the default window option for many releases now. 

It might be that some other code in OF is relying on the GLUT library, but we could definitely start a PR to remove it and see what passes.
